Ladybird/WebContentView.cpp

```diff
@@ -322,6 +324,19 @@ void WebContentView::mouseReleaseEvent(QMouseEvent* event)
     client().async_mouse_up(to_content(position), button, buttons, modifiers);
 }
 
+void WebContentView::dragEnterEvent(QDragEnterEvent* event)
+{
+    if (event->mimeData()->hasUrls())
+        event->acceptProposedAction();
+}
+
+void WebContentView::dropEvent(QDropEvent* event)
+{
+    VERIFY(event->mimeData()->hasUrls());
+    load(ak_deprecated_string_from_qstring(event->mimeData()->urls()[0].toString()));
```

Comment:
Wouldn't it make more sense to load the remaining URLs in a new tab? At least that's what Firefox does.